clients/adapter-amqp/src/main/java/org/eclipse/hono/adapter/client/amqp/AbstractRequestResponseServiceClient.java

```diff
@@ -0,0 +1,369 @@
+/*******************************************************************************
+ * Copyright (c) 2016, 2020 Contributors to the Eclipse Foundation
```

Comment:
2020

Reply:
I had deliberately kept the origin year because most of the code has been copied over from the legacy client.